### PR TITLE
Refactor pipe and rename pipe monitor

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

- Add a "Mode" to Pipe `__init__`
- Rename `PipeMonitor` to `PipeHandler`

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/fuel/utils/pipe/pipe.py [hunk 1]
```diff
@@ -15,11 +15,17 @@
 import re
 import uuid
 from abc import ABC, abstractmethod
+from enum import Enum
 from typing import Any, Union
 
 from nvflare.fuel.utils.validation_utils import check_str
 
 
+class Mode(str, Enum):
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/fuel/utils/pipe/pipe.py:24]: Ok, let me see how can I change

**Comment by yhwen** [comment]
I did the test of the FL-Hub functions with Pipe changes. FL-Hub functions work as expected.

**Review by nvidianz — APPROVED** [review]

**Pull request merged**